acceptance-tests/FamilyHubs.ServiceDirectory.Api.AcceptanceTests/Tests/Stories/ServiceSearchTests.cs

```diff
@@ -1,95 +1,183 @@
+using System.Net;
 using TestStack.BDDfy;
 using FamilyHubs.ServiceDirectory.Api.AcceptanceTests.Tests.Steps;
+using Xunit;
 
 namespace FamilyHubs.ServiceDirectory.Api.AcceptanceTests.Tests.Stories;
 
 // Define the story/feature being tested
 [Story(
-   AsA = "user of the metrics api",
-   IWant = "to be able to update usage information for postcode searches and filters",
-   SoThat = "I can record how search functionality is being used")]
-
+    AsA = "user of the metrics api",
+    IWant = "to be able to update usage information for postcode searches and filters",
+    SoThat = "I can record how search functionality is being used")]
 [TestClass]
 public class ServiceSearchTests
 {
-   private readonly ServiceSearchSteps _steps;
+    private readonly ServiceSearchSteps _steps;
+
+    public ServiceSearchTests()
+    {
+        //Get instances of the steps required for the test
+        _steps = new ServiceSearchSteps();
+    }
+
+    //Add all tests that make up the story to this class.
+    [Theory]
```

Comment:
(As discussed on a call)

refactor: The advantage of a `[Theory]` is that you can add tests without duplicating the code, so you could have something like:

```
[Theory]
[InlineData("15", "E1 2EN", "200", "1", "2", HttpStatusCode.OK)] // Find, Initial Search
[InlineData("15", "E1 2EN", "200", "1", "1", HttpStatusCode.OK)] // Connect, Initial Search
public void ServiceSearch_Metric_Endpoint_Returns_200_Success(...) { ... }
```

And it will run that test multiple times for each input defined in `InlineData` :)

Reply:
Might be worth as well ditching the `expectedStatusCode` input and just putting `ThenExpectedStatusCodeReturned(HttpStatusCode.OK)` in the test, to better show the intent of the title, and then you can repeat theories for returning 400 codes and such